framework/scenario/src/scenario/parse_util.rs

```diff
@@ -16,7 +18,8 @@ pub fn parse_scenario_raw<P: AsRef<Path>>(path: P) -> ScenarioRaw {
 
 pub fn parse_scenario<P: AsRef<Path>>(path: P) -> Scenario {
     let scenario_parent = path.as_ref().parent().unwrap();
-    let interpreter_context = InterpreterContext::new(scenario_parent.into());
+    let interpreter_context =
```

Comment:
Get rid of the `new` with arguments.
Leave a `new` that is the same as the default.
`InterpreterContext::new().with_dir(scenario_parent.into())`